src/manage-apps/manage-repos.md

```diff
@@ -0,0 +1,78 @@
+# Managing Repositories
+
+Add, remove, and edit repositories using Repoman. Repoman allows users to add Personal Package Archives (PPAs) and Flatpak repositories. Users may want to add a PPA or additional Flatpak repository if their software is not included in the Pop!\_Shop, or they may want to remove PPA’s after uninstalling a package.
```

Comment:
I thought I'd commented on this before, but apparently not (maybe it was in the Google Doc.) PPAs are not the only deb repositories out there, they are just deb repositories hosted by Canonical (the sponsors of Ubuntu) on Launchpad. "PPA" is essentially a brand name for the deb repositories that Canonical hosts. It's usually more appropriate to refer to "deb repositories"/"Deb repositories"/"Debian repositories" (however you want to style it) than "PPAs."

Reply:
I recall that feedback and had attempted to implement it here. This is probably just my own knowledge gap. I assumed PPA's were only used for apps not hosted in "official" Debian repos.

Reply:
> I assumed PPA's were only used for apps not hosted in "official" Debian repos.

"Official Debian repos" sounds like the repos that the Debian distribution is using, which we never use.

The official Ubuntu and Pop!_OS repos, which serve packages in Debian format, are what you mean. Maybe calling it an "apt repository" since it's the format that apt uses would be less prone to confusion. Third-party apt repos can be PPAs or they can just be apt repositories hosted elsewhere, which are exactly the same thing except hosted somewhere other than Launchpad.

That last sentence is what I was trying to get across; PPAs are a subset of apt repos, so the two terms are not synonymous, and it's better to use the broader term unless you're specifically talking about Canonical-hosted Launchpad PPAs. You are correct that the use case for third-party repos of any kind are for apps not hosted in the official, included Pop!_OS or Ubuntu repos.

Reply:
I've changed all mentions of "Debian repository" to "apt repository". I've also added introductory content to manage-repos.md.